official/resnet/ctl/ctl_imagenet_benchmark.py

```diff
@@ -33,7 +33,50 @@
 FLAGS = flags.FLAGS
 
 
-class Resnet50CtlAccuracy(ctl_benchmark.CtlBenchmark):
+class CtlBenchmark(PerfZeroBenchmark):
+  """Base benchmark class with methods to simplify testing."""
+
+  def __init__(self, output_dir=None, default_flags=None, flag_methods=None):
+    self.output_dir = output_dir
+    self.default_flags = default_flags or {}
+    self.flag_methods = flag_methods or {}
+    super(CtlBenchmark, self).__init__(
+        output_dir=self.output_dir,
+        default_flags=self.default_flags,
+        flag_methods=self.flag_methods)
+
+  def _report_benchmark(self,
+                        stats,
+                        wall_time_sec,
+                        top_1_max=None,
+                        top_1_min=None):
+    """Report benchmark results by writing to local protobuf file.
+
+    Args:
+      stats: dict returned from keras models with known entries.
+      wall_time_sec: the during of the benchmark execution in seconds
+      top_1_max: highest passing level for top_1 accuracy.
+      top_1_min: lowest passing level for top_1 accuracy.
+    """
+
+    metrics = []
+    if 'eval_acc' in stats:
+      metrics.append({'name': 'accuracy_top_1',
+                      'value': stats['eval_acc'],
+                      'min_value': top_1_min,
+                      'max_value': top_1_max})
+      metrics.append({'name': 'top_1_train_accuracy',
```

Comment:
this seems wrongly setting loss value for accuracy metric?

Reply:
ah yes. reworked this a bit.